packages/smart-wallet/test/test-psm-integration.js

```diff
@@ -155,108 +156,148 @@ test('want stable', async t => {
   const { anchor } = t.context;
-  const { agoricNames, economicCommitteeCreatorFacet, psmFacets, zoe } =
+  const { agoricNames, /* economicCommitteeCreatorFacet, */ psmFacets, zoe } =
     await E.get(t.context.consume);
 
-  const anchorPsm = await E.get(E(psmFacets).get(anchor.brand));
-
-  const { psmGovernorCreatorFacet } = anchorPsm;
+  const { psm: psmInstance } = await E(psmFacets).get(anchor.brand);
 
   const wallet = await t.context.simpleProvideWallet(committeeAddress);
   const computedState = coalesceUpdates(E(wallet).getUpdatesSubscriber());
   const offersFacet = wallet.getOffersFacet();
 
-  t.log('Deposit voter invitation into wallet');
-  {
-    const invitations = await E(
-      economicCommitteeCreatorFacet,
-    ).getVoterInvitations();
-    const voterInvitation = await invitations[0];
-    t.assert(
-      await E(E(zoe).getInvitationIssuer()).isLive(voterInvitation),
-      'invalid invitation',
-    );
-    wallet.getDepositFacet().receive(voterInvitation);
-  }
-
-  t.log('Set up question');
-  const binaryVoteCounterInstallation = await E(agoricNames).lookup(
-    'installation',
-    'binaryVoteCounter',
-  );
-  const { details } = await E(psmGovernorCreatorFacet).voteOnOfferFilter(
-    binaryVoteCounterInstallation,
-    2n,
-    harden(['wantStable']),
+  const psmCharter = await E(agoricNames).lookup('instance', 'psmCharter');
+  const economicCommittee = await E(agoricNames).lookup(
+    'instance',
+    'economicCommittee',
   );
-  const { positions, questionHandle } = await details;
-  const yesFilterOffers = positions[0];
+  await eventLoopIteration();
 
-  t.log('Prepare offer to voting invitation in purse');
-  {
-    // get invitation details the way a user would
-    const invitationDetails = await E(E(zoe).getInvitationIssuer())
+  // get invitation details the way a user would
+  const getInvitationFor = async (desc, len, balances) =>
+    E(E(zoe).getInvitationIssuer())
       .getBrand()
       .then(brand => {
         /** @type {Amount<'set'>} */
-        const invitationsAmount = NonNullish(computedState.balances.get(brand));
-        t.is(invitationsAmount?.value.length, 1);
-        return invitationsAmount.value[0];
+        const invitationsAmount = NonNullish(balances.get(brand));
+        t.is(invitationsAmount?.value.length, len);
+        return invitationsAmount.value.filter(i => i.description === desc);
       });
 
-    /** @type {import('../src/invitations.js').PurseInvitationSpec} */
-    const invitationSpec = {
-      source: 'purse',
-      instance: await E(agoricNames).lookup('instance', 'economicCommittee'),
-      description: invitationDetails.description,
-    };
-    /** @type {import('../src/offers').OfferSpec} */
-    const offerSpec = {
-      id: 33,
-      invitationSpec,
-      proposal: {},
-    };
-    t.log('Execute offer for the invitation');
-    await offersFacet.executeOffer(offerSpec);
-  }
+  /** @type {{description: string, instance: Instance}} */
```

Comment:
better to put this on the return type of `getInvitationFor` so it's clear what it's supposed to return